databases/week1/w1.sql

```diff
@@ -0,0 +1,17 @@
+SELECT COUNT(task.id) FROM task;
```

Comment:
You should create a new line per statement, e.g.
`SELECT COUNT(task.id)`
`FROM task;`

Reply:
I hope I edited this part correctly.